site/src/docs/backup/migration/index.md

```diff
@@ -3,3 +3,30 @@ title: Migration
 parent: Backup
 order: 300
 ---
+
+Here is an example on how to move your comments after your posts are moved, from `https://example.org/blog/<slug>` to `https://example.org/post/<slug>` in that example.
+
+### Rules file
+
+First you have to create a `rules` file in remark's `/var` folder.
+
+[Here is the test](https://github.com/akosourov/remark/blob/4dc123dbe84f4f248864bcdbbd6cc2b3a4dafe11/backend/app/migrator/mapper_test.go#L13-L17) with an example of rules file syntax, format is simply `old_url new_url` like following:
```

Comment:
I don't think it's a good idea to mention code. I think all of the information should be shown on the doc page.